tests/ui/type-alias-impl-trait/hkl_forbidden2.stderr

```diff
@@ -0,0 +1,5 @@
+error[E0792]: expected generic lifetime parameter, found `'a`
```

Comment:
:thinking: please figure out why we use a dummy span here :sweat_smile: we should point to the opaque :sweat_smile: